### PR TITLE
Added Hindi translation strings

### PR DESCRIPTION
**Pull request by ayushi790**
Added hindi translation strings as a part of [#462](https://github.com/android/sunflower/issues/462).
Below are some screenshots for the reference.

WhatsApp Image 2021-06-27 at 15 28 36 (1)

WhatsApp Image 2021-06-27 at 15 28 36

### CODE REVIEW

File: app/src/main/res/values-hi/strings.xml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<resources>
+    <!-- Translatable resources -->
+    <string name="menu_filter_by_grow_zone">ग्रो ज़ोन द्वारा फ़िल्टर करें</string>
+    <string name="plant_detail_image_content_description">पेड़ों की तस्वीरें</string>
+    <string name="my_garden_title">मेरा बाग</string>
```
Comment (Umesh-01) [inline app/src/main/res/values-hi/strings.xml:21]: Change `मेरा बाग` to `मेरा बगीचा`

File: app/src/main/res/values-hi/strings.xml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+    <!-- String plurals and their related prefix / suffix strings -->
+    <string name="watering_needs_prefix">पानी की जरूरत</string>
+    <plurals name="watering_needs_suffix">
+        <item quantity="one">रोज रोज</item>
+        <item quantity="other">हर %d दिन</item>
+    </plurals>
+
+    <plurals name="watering_next">
+        <item quantity="one">कल पानी की जरूरत है</item>
```
Comment (Umesh-01) [inline app/src/main/res/values-hi/strings.xml:41]: Change `है` to `होगी`
OR
Change `कल पानी की जरूरत है।` to `कल पानी देने की जरूरत है।`

File: app/src/main/res/values-hi/strings.xml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    <string name="plant_date_header">बुवाई पूर्ण</string>
+    <string name="watered_date_header">ताजा पानी दो</string>
+    <string name="menu_item_share_plant">शेयर</string>
+    <string name="share_text_plant">Android Sunflower ऐप में %s ट्री देखें</string>
+
+    <!-- String plurals and their related prefix / suffix strings -->
+    <string name="watering_needs_prefix">पानी की जरूरत</string>
+    <plurals name="watering_needs_suffix">
+        <item quantity="one">रोज रोज</item>
+        <item quantity="other">हर %d दिन</item>
```
Comment (Umesh-01) [inline app/src/main/res/values-hi/strings.xml:37]: Change `हर %d दिन` to `प्रत्येक %d दिन बाद`

**Review by Umesh-01 — COMMENTED** [review]
@ayushi790 do the following changes..

**Comment by riggaroo** [comment]
Hi, 
Thanks for your contribution, however Sunflower is going to be archived in favour of maintaining a smaller set of up-to-date set of compose-samples, that can be found here: https://github.com/android/compose-samples, as well as snippets that can be found here: https://github.com/android/snippets.

Please feel free to keep your fork up-to-date for your learning, but this repository will be archived shortly.

**Pull request closed without merging**